### PR TITLE
Mh/update user facing text

### PR DESCRIPTION
**Pull request by mhenschke-atl**
Copy updated to reflect the changes in the mural

### CODE REVIEW

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div>
-    Enter the repository slug. The slug can be found in the URL after the repos/ line. **Note** this may be different
-    to the *repo name* listed on the page at your.bitbucket.url.com/project/YOURPROJECTKEY/repos/
+    This is the repository name that is used in the repository URL. For example, for the repository URL
```
Comment (khughes-atlassian) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html]: I don't think we should call the repo slug the 'repository name' because that can be confusing.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
-bitbucket.admin.token=Admin token
+bitbucket.admin.token=Personal token
```
Comment (khughes-atlassian) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties]: I think this said 'Admin' token because we require a personal token from an administrator. Changing this text to just 'Personal' token is a bit confusing because we loose the bit that explains it has to be from an admin

Comment (jvanderloo) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties]: How about "Administrator's personal token" or "Personal token with admin privileges" 
(btw is it repo or instance admin?)

Is there somewhere else to call out the permission level if it doesn't make sense to make this field more descriptive?

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html (hunk 1 above)
Comment (jvanderloo) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html]: If the field is called "repository slug" or similar and we wanted to explain it here perhaps we could rephrase this to say something like "The slug is the URL-safe version of the repository name."?

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
+    'https://bitbucket.org/teamsinspace/documentation-tests', the slug is 'documentation-tests'.
```
Comment (jvanderloo) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html]: Should we use a not `bitbucket.org` example here?

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-serverId.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
 <div>
-    Select a Bitbucket Server from the list. If you cannot see your server listed, check the System Configuration
-    (Manage Jenkins -> Configure System).
+    This is the Bitbucket instance that contains the project you want Jenkins to build from. If you cannot see your
```
Comment (jvanderloo) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-serverId.html]: /nit "contains the _repository_" 
(because projects are just a container for the things that Jenkins actually needs access to)

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-serverId.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+    instance check your credentials and try again.
```
Comment (jvanderloo) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-serverId.html]: /nit "check your configuration" ? 

I'm thinking there could be other reasons that the instance might not be listed, so perhaps being more generic and asking to check config rather than _just_ credentials might be better.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/config.properties [hunk 6]
```diff
@@ -1,8 +1,8 @@
 bitbucket=Bitbucket
 bitbucket.admin.credentials=Project admin token
-bitbucket.server.name=Server name
-bitbucket.url=URL
+bitbucket.server.name=Instance name
+bitbucket.url=Instance URL
 bitbucket.credentials=Credentials
-bitbucket.test.connection=Test connection
+bitbucket.test.connection=Check credentials
```
Comment (jvanderloo) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/config.properties]: /nit IMO "Test connection" is better because we're also validating the instance URL, not just the credentials

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-credentialsId.html [hunk 7]
```diff
@@ -1,5 +1,7 @@
 <div>
-    The list of credentials used to authenticate with Bitbucket Server.<br>
-    Bitbucket Server supports username and password credentials, as well as secret word credentials. When adding new
-    credentials, only these types should be used (other credential types will not appear in the list.)
+    Adding Bitbucket project-access credentials is not mandatory, but it makes it easier for users to set up Jenkins projects.<br><br>
```
[the remark this answers is not in the document]
Comment (dkjellin) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-credentialsId.html]: I know we said we'd support cloning by token, however, I think we should remove any mention of tokens until we know it works.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties (hunk 2 above)
Comment (dkjellin) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties]: Project admin, you can't create "instance admin" through the UI.

Comment (jvanderloo) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties]: Sorry when I said instance admin, I meant should the token belong to someone that *is* an instance admin, because they should have read access to all the projects on the instance.

**Comment by mhenschke-atl** [comment]
Update on this: The copy for the server and plugin configuration has been updated with the correct mural, but SCM is still a WIP so no need to review that yet.

**Comment by ctan-atlassian** [comment]
Hi @dkjellin. Following on from our chat about linking to the BbS docs. Taking the personal access tokens page as an example, if you use this link it will always redirect to the latest version of the documentation. 

> https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties (hunk 2 above)
Comment (dkjellin) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/config.properties]: Hmm, how about 'Token value` or `Token text` 
At this point we are indicating what the field input is, so maybe even just `Token` is enough?

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html [hunk 8]
```diff
@@ -1,5 +1,7 @@
 <div>
-    The list of credentials used to authenticate with Bitbucket Server.<br>
-    Bitbucket Server supports username and password credentials, as well as secret word credentials. When adding new
-    credentials, only these types should be used (other credential types will not appear in the list.)
+    Credentials are required for Jenkins to authenticate with Bitbucket. Depending on the credentials used, they give Jenkins
+    access to some or all of the repositories on your Bitbucket instance, and allow it to create builds. To provide them:<br>
+    1. Select credentials for your instance from the dropdown list (if they exist)<br>
+    2. Add your own credentials for your instance<br>
+    The credentials you provide will determine which Bitbucket instances appear below.
```
Comment (dkjellin) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html]: That is not entirely correct. The dropdown is always populated with the entire list of servers, the credentials _may_ or _may not_ work for a particular instance. 

I think this whole text needs a slight polish as it is not entirely clear how things hang together. How about
```html
Credentials are required for Jenkins to authenticate with Bitbucket. Depending on the credentials used, they give Jenkins access to some or all of the repositories on your Bitbucket instance, and allow it to check out source code for building. To provide them:<br>
<ul>
    <li>
        Select credentials for your instance from the dropdown list (if they exist).
        <br>
        or
    </li>
    <li>Use the credentials provided for the instance (select no credentials to fall back to the instance defined credentials should they exist).</li>
   
</ul>
The credentials you provide will determine which projects and repositories can be cloned.
```

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-baseUrl.html [hunk 9]
```diff
@@ -1,3 +1,3 @@
 <div>
-    The URL where Bitbucket is located
+    This is the URL that you'd use in your browser to get to the Bitbucket instance. E.g.: https://bitbucketserver-url.com
```
[the remark this answers is not in the document]
Comment (dkjellin) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-baseUrl.html]: Should we mention that the `/dasboard` should not be included?

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html (hunk 1 above)
Comment (ctan-atlassian) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html]: Following what the [BBC docs say about repository slugs ](https://confluence.atlassian.com/bitbucket/what-is-a-slug-224395839.html) I've changed this to:
> This is the URL-friendly version of the repository name. 
Thanks

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-credentialsId.html [hunk 10]
```diff
@@ -1,5 +1,7 @@
...
+    Project-access credentials allow Jenkins to authenticate with a Bitbucket instance and retrieve the projects on it.
+    They will be able to select and use them to give Jenkins access to their project.<br><br>
```
[the remark this answers is not in the document]
Comment (ctan-atlassian) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-credentialsId.html]: This text has been revised.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html [hunk 11]
```diff
@@ -1,5 +1,6 @@
 <div>
-    A personal access token with project admin permission. This is used for things that require elevated permissions
-    (such as creating webhooks).
-    This token is not visible to users, and is not used to access projects and/or repositories.
+    The personal token must have project admin permissions.<br><br>
+    it allows users to automatically set up build triggers when creating a Jenkins project.<br><br>
+    Users cannot access these credentials<br><br>
+    It needs to be a Bitbucket Project Admin Token.<br><br>
```
[the remark this answers is not in the document]
Comment (ctan-atlassian) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html]: This text has been revised.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-baseUrl.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (ctan-atlassian) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-baseUrl.html]: I've made this:

>This is the URL you'd use in your browser to get to this Bitbucket instance, edited to remove _/dashboard_. E.g.: https://bitbucketserver-url.com

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html (hunk 8 above)
Comment (ctan-atlassian) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html]: Thanks @dkjellin . I've made some updates for the next round of UI changes.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-description.html [hunk 12]
```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>This description will help users identify this token when setting up build triggers</p>
```
Comment (dkjellin) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-description.html]: Technically incorrect. "This description will help administrators identify this token when configuring a Bitbucket server instance"

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html [hunk 13]
```diff
@@ -1,5 +1,7 @@
 <div>
-    A personal access token with project admin permission. This is used for things that require elevated permissions
-    (such as creating webhooks).
-    This token is not visible to users, and is not used to access projects and/or repositories.
+    <p>Adding a <a href="https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens">Bitbucket
+        personal access token</a> allows users to automatically set up build triggers when creating a Jenkins project.
+        Importantly, for this to work the token must have project admin permissions.</p>
+
+    <p>Users will only be able to use the token to set up build triggers.</p>
```
Comment (dkjellin) [inline src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html]: I think this text is wrong here. If I come in and I wonder "what does this 'token' field need?" This text is not helping me. This text is for the **field** so I think we should be far more specific 
```html
<p>This is the value of the token that was displayed when the token was created in Bitbucket Server</p>
```
or something similar.

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-description.html (hunk 12 above)
Comment (mhenschke-atl) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-description.html]: @ctan-atlassian just going to make this change

File: src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html (hunk 13 above)
Comment (mhenschke-atl) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html]: I've addressed this by kind of mixing them together- preserving the hyperlink and inserting that into your sentence, and keeping the following text.

Comment (ctan-atlassian) [reply src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html]: Hi @dkjellin.  You're right and it should be more of an instruction. The only thing I'm not sure about is whether we need to say that the token value is displayed when the token is created. 

In the docs we recommend using one token per integration. I think we should mention this here. This means that telling users when the token value is displayed isn't needed. I also think it's worth mentioning the benefits again in case they don't read the first help text. 

I think we should change this to: 

> Here you should enter a [personal access token] https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens) with project admin permissions that you created in Bitbucket. This will allow users to automatically set up build triggers when creating a Jenkins project. 
> 
> We recommend creating a new token to use in this plugin. This way, if this token needs to be removed no other plugins will be affected. 
>
>Users will only be able to use the token you add to set up build triggers.
